tests/e2e/api/collection.test.ts

```diff
@@ -0,0 +1,39 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+import { Aptos, AptosConfig, Network } from "../../../src";
+
+describe("Collection", () => {
+  test("it should get collection data", async () => {
+    const config = new AptosConfig({ network: Network.MAINNET });
```

Comment:
that's a great test, I think we can skip test it for now until we have a local support for creating collection and then test against localnet